app/replica.go

```diff
@@ -128,13 +128,8 @@ func startReplica(c *cli.Context) error {
 	}
 
 	dir := c.Args()[0]
-	backingFile, err := openBackingFile(c.String("backing-file"))
```

Comment:
@utkarshmani1997 @prateekpandey14 @payes -- why was this Backfile used? What is the impact of removing this?